--- src/usr/share/applications/chromium.desktop	2022-12-08 16:05:28.611393312 +0700
+++ src/usr/share/applications/chromium.new.desktop	2022-12-08 16:05:57.141392635 +0700
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-Name=Chromium
+Name=JIΛweb
 # Only KDE 4 seems to use GenericName, so we reuse the KDE strings.
 # From Ubuntu's language-pack-kde-XX-base packages, version 9.04-20090413.
 GenericName=Web Browser
@@ -108,7 +108,7 @@
 Exec=/usr/bin/chromium %U
 StartupNotify=true
 Terminal=false
-Icon=chromium
+Icon=/usr/share/icons/cvm-ui-icons/jiaweb.png
 Type=Application
 Categories=Network;WebBrowser;
 MimeType=application/pdf;application/rdf+xml;application/rss+xml;application/xhtml+xml;application/xhtml_xml;application/xml;image/gif;image/jpeg;image/png;image/webp;text/html;text/xml;x-scheme-handler/http;x-scheme-handler/https;
