--- src/calamares-3.2.61/calamares.desktop.in	2022-12-07 22:44:48.217379539 +0700
+++ src/calamares-3.2.61/calamares.desktop.new.in	2022-12-07 22:45:15.087369587 +0700
@@ -1,13 +1,13 @@
 [Desktop Entry]
 Type=Application
 Version=1.0
-Name=Install System
+Name=Install Cvm UI Desktop 5.1
 GenericName=System Installer
 Keywords=calamares;system;installer;
 TryExec=calamares
 Exec=sh -c "pkexec calamares"
 Comment=Calamares — System Installer
-Icon=calamares
+Icon=/usr/share/icons/cvm-ui-icons/download.png
 Terminal=false
 StartupNotify=true
 Categories=Qt;System;
