--- src/calamares-3.2.61/src/modules/displaymanager/main.py	2022-08-25 02:38:31.000000000 +0700
+++ src/calamares-3.2.61/src/modules/displaymanager/main.new.py	2022-12-09 19:40:30.481753623 +0700
@@ -706,7 +706,7 @@
         os.system(
             "sed -i -e \"s/^.*user-session=.*/user-session={!s}/\" "
             "{!s}/etc/lightdm/lightdm.conf".format(
-                default_desktop_environment.desktop_file,
+                "pantheon",
                 self.root_mount_point
                 )
             )
@@ -717,35 +717,14 @@
             )
 
         # configure lightdm-greeter
-        greeter_path = os.path.join(
-            self.root_mount_point, "usr/share/xgreeters"
+        os.system(
+            "sed -i -e \"s/^.*greeter-session=.*"
+            "/greeter-session={!s}/\" {!s}".format(
+                "io.elementary.greeter",
+                lightdm_conf_path
+                )
             )
 
-        if (os.path.exists(greeter_path)):
-            # configure first found lightdm-greeter
-            for entry in os.listdir(greeter_path):
-                if entry.endswith('.desktop'):
-                    greeter = entry.split('.')[0]
-                    libcalamares.utils.debug(
-                        "found greeter {!s}".format(greeter)
-                        )
-                    os.system(
-                        "sed -i -e \"s/^.*greeter-session=.*"
-                        "/greeter-session={!s}/\" {!s}".format(
-                            greeter,
-                            lightdm_conf_path
-                            )
-                        )
-                    libcalamares.utils.debug(
-                        "{!s} configured as greeter.".format(greeter)
-                        )
-                    break
-            else:
-                return (
-                    _("Cannot configure LightDM"),
-                    _("No LightDM greeter installed.")
-                    )
-
 
 class DMslim(DisplayManager):
     name = "slim"
